intel_extension_for_transformers/backends/neural_engine/graph/models/gptj/gptj.cpp

```diff
@@ -131,7 +154,16 @@ static bool gptj_model_eval_internal(model_context& lctx, const model_token* tok
                                                       1 * N * n_embd * batch_size * ne_element_size(QKVcur)),
                                            n_embd / n_head, n_head, N, batch_size),
                              n_past, n_rot, 0);
-      Vcur = ne_view_1d(ctx0, QKVcur, N * n_embd * batch_size, 2 * N * n_embd * batch_size * ne_element_size(QKVcur));
+      if (!kv_mem_jblas) {
+        Vcur = ne_view_1d(ctx0, QKVcur, N * n_embd * batch_size, 2 * N * n_embd * batch_size * ne_element_size(QKVcur));
+      } else {
+        Vcur = ne_rope_inplace(ctx0,
```

Comment:
`Vcur` does not need `rope_embeddings`

Reply:
Thanks. It has been fixed in 90e51f6 and the performance data in the PR description has been updated accordingly.